test/packages/caver.klay.accounts.js

```diff
@@ -799,6 +799,375 @@ describe('caver.klay.accounts.encrypt', () => {
       expect(() => caver.klay.accounts.encrypt(testAccount.getKlaytnWalletKey(), password, {address: caver.klay.accounts.create().address})).to.throw(errorMessage)
     })
   })
+
+  context('CAVERJS-UNIT-WALLET-351: input: array of private key string, password, {address:valid}', () => {
```

Comment:
Nice test cases!! 
It would be better if we have:
1. test hard-coded keystore JSON encryption for our new format.
2. test hard-coded keystore JSON decryption for our new format.
3. original web3 keystore JSON format loading

Reply:
I added test cased for 2, 3.
However, in case 1, the result of encrypt changes every time encrypt is performed, so the only values that can be compared when comparing expected json results are fixed values (address and version).
That's why I didn't add test code for case 1.

Reply:
What makes the change? If it is because of new private key, we can put a sample private key.

Reply:
The parameters used for encrypt (optional values you can set in options) make a difference. If you think this test case is meaningful, i will try to test it by setting all the parameter values.

Reply:
The test case for 1 is added :)